.fttemplates/MDXStories/.ftsettings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "omitParentDirectory": true
```

Comment:
if we are keeping them can the file and folder names align more with our conventions (lower kebab case)? 

`.fttemplates/mdx-stories/<ftname>.mdx`

Reply:
Removed the folder templates and tooling recommendation 👍